openapi/src/main/openapi/common/components.yaml

```diff
@@ -68,6 +68,103 @@ schemas:
     example:
       customProperty: "value"
 
+  StopCriteria:
+    type: array
+    description: An array of stop criteria that includes CPU, error ratio, and response time configurations.
```

Comment:
They are used to automatically stop a run if the threshold is reached over the rolling timeframe 

Shall we add this or is it not the place ? Not sure